### DISCUSSION TITLE
Is the application page buggy?

### DISCUSSION THREAD

**Discussion by fannika12** [community/community#128836] [GitHub Education]
### Hi everyone,

Hi everyone,

I'm having some trouble with my GitHub Education application, and I wanted to share my experience to see if anyone else has encountered similar issues or has any advice.

Here's what happened:

I received a verification email to my university email, which I accepted.
I uploaded a photo of my student ID. The front of my ID contains the name of my school, my name, and my photo, but only the back of the ID has the validity date.
I proceeded with the application, but almost immediately, I received an email to my student email account stating that my application was rejected. The reasons were:
I need to enable 2FA (which I have now done).
I need to set up billing info (which I have also done).
I need to fill out my profile (also completed).
Additionally, the email mentioned that the photo I uploaded of my student ID doesn't seem to contain my school name, even though it is on the front of the ID. I didn't find a way to upload multiple photos at once, and it seems to let me upload an infinite number of images without any indication of error or success.

Has anyone else faced this issue? How did you manage to get past it? Any tips or advice would be greatly appreciated!

Thanks in advance for your help!

**Comment by davevad93** [comment]
Hi @fannika12 ,

Regarding the student ID I suggest you take a photo of both sides, combine them into one picture and then upload it. That usually solves the ID related rejection issue. About the other reasons for rejection it's a bit weird since you have already done everything they ask for, so try making another request by uploading your ID photo as I told you, maybe that will fix it.

**Comment by BastiCoding** [comment]
I habe the Same issue i get an Error that my Name in the Billing Info doesnt Match the document which i submitted while they are exactly the Same to the character. I tried multipel Times to reaply but the Error stays

**Comment by javawolfpack** [comment]
I keep submitting my ID w/ a URL to my department page, which has worked for years when I've had to reapply. I get nothing, page just blanks out and asks me to submit again... my contacts at GitHub education seem to have left so can't seem to get help on this. 

Mind you my profile shows I'm still recognized but keeps revoking my orgs when I give them access.
